modules/ocf/templates/firefox/prefs.js.erb

```diff
@@ -1,3 +1,25 @@
+// This is the Debian specific preferences file for Firefox ESR
+// You can make any change in here, it is the purpose of this file.
+// You can, with this file and all files present in the
+// /etc/firefox-esr directory, override any preference you can see in
+// about:config.
+//
+// Note that pref("name", value, locked) is allowed in these
+// preferences files if you don't want users to be able to override
+// some preferences.
+
+pref("extensions.update.enabled", true);
```

Comment:
All extensions live in `~/.config`, which gets cleared on logout. While this doesn't hurt us, it's kinda irrelevant since extensions are per-session and hence are always up-to-date. This setting is targeted at personal computers.